README.md

```diff
@@ -1,6 +1,7 @@
 ## Oscillate Network
 
-Copyright (c) 2018-2019 The Oscillate Network.    
+Copyright (c) 2018-2019 The Oscillate Network.
```

Comment:
Should not be only 2019 year stated?

Reply:
...that would make sense, as we dont have any time travelers :)